benchmark-service/benchmark/benchmark.py

```diff
@@ -28,21 +73,51 @@ async def greet_user(request: web.Request) -> web.Response:
         'username': request.match_info.get('username', ''),
         'current_date': 'July 10, 2020'
     }
-    response = aiohttp_jinja2.render_template('user.html', request,
+    response = aiohttp_jinja2.render_template('name.html', request,
                                               context=context)
     return response
 
 
+@router.get('/name/{name}')
+@web_authenticated_developers_only(redirect=False)
+async def show_name(request: web.Request, userdata) -> web.Response:  # pylint: disable=unused-argument
+    benchmarks = get_benchmarks(default_filepath)
```

Comment:
I changed my mind. I think the goal should be to show the benchmarks for any arbitrary file here. @danking Do you have suggestions on how to pass the file name in this GET request?